client/ayon_core/cli_commands.py

```diff
@@ -73,6 +73,18 @@ def publish(paths, targets=None, gui=False):
         import pyblish.api
         import pyblish.util
 
+        # Fix older jobs
+        for src_key, dst_key in (
+            ("AVALON_PROJECT", "AYON_PROJECT_NAME"),
+            ("AVALON_ASSET", "AYON_FOLDER_PATH"),
+            ("AVALON_TASK", "AYON_TASK_NAME"),
+            ("AVALON_WORKDIR", "AYON_WORKDIR"),
+            ("AVALON_APP_NAME", "AYON_APP_NAME"),
+            ("AVALON_APP", "AYON_HOST_NAME"),
+        ):
+            if src_key in os.environ and dst_key not in os.environ:
+                os.environ[dst_key] = os.environ[src_key]
```

Comment:
Do we then also want to pop out the original key - just to ensure that even for older jobs we are actually using only the new var? (Not sure if really needed)

Reply:
Hmm, tru, maybe we should. Will add.